archiver_test.go

```diff
@@ -73,7 +73,7 @@ func (s *ArchiverSuite) SetupTest() {
 	s.tx = rrepository.NewSivaRootedTransactioner(rrepository.NewLocalCopier(s.rootedFs, s.bucket), s.txFs)
 
 	ls, err := lock.NewLocal().NewSession(&lock.SessionConfig{
-		Timeout: time.Second * 1,
+		Timeout: time.Minute * 1,
```

Comment:
we should use `defaultTimeout` in here instead of defining it before and then here again